### ISSUE TITLE
too many accounts = error too many requests

### ISSUE THREAD

**Issue by ConteXCrown** [Jessecar96/SteamDesktopAuthenticator#768]
Hi,
when you have multiple accounts set up in the SDA the Auto Accepts doesnt go through, because the App will request f. e. 8 requests at the same time and Steam will  send you on to the bot protection

you need like a delay between each account or some
![s](https://user-images.githubusercontent.com/98663891/180095936-2bffe224-f2be-4eb7-a8d0-ba041912857e.PNG)
.

**Comment by pessutuliy** [comment]
yes, exactly, it is, if you separate the accounts everything works
THX
